_data/current.yml

```diff
@@ -1,3 +1,10 @@
+- name: Odin Testkonferansen Norway 2024
+  location: Oslo
+  dates: "September 26, 2024"
+  url: www.https://event.dataforeningen.no/odin/
```

Comment:
This looks like a typo

Reply:
Hi, what looks like a type ?
Is it the - before mane?

Reply:
the url is: `www.https://` and that looks like a typo. Remove the www.